app/src/main/java/org/fossasia/openevent/general/social/SocialLinksDao.kt

```diff
@@ -0,0 +1,20 @@
+package org.fossasia.openevent.general.social
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import io.reactivex.Flowable
+import org.fossasia.openevent.general.event.Event
+
+@Dao
+interface SocialLinksDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertSocialLinks(socialLinks: List<SocialLink>)
+
+    @Query("DELETE FROM SocialLink")
+    fun deleteAll()
+
+    @Query("SELECT * from SocialLink ORDER BY startsAt DESC")
+    fun getAllSocialLinks(id: Long): Flowable<List<SocialLink>>
```

Comment:
Wrong logic, should get social links for an event

Reply:
Yeah Actually I haven't implemented db storage yet, I am doing it :+1: